### PR TITLE
ORM_homework

### PR DESCRIPTION
**Pull request by akarev0**

### CODE REVIEW

File: Flask_ORM/many_to_many/routes.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    @marshal_with(staff_structure)
+    def get(self):
+        return StaffModel.query.all()
+
+    def post(self):
+        data = json.loads(request.data)
+        new_staff = StaffModel(**data)
+        db.session.add(new_staff)
+        db.session.commit()
+        return "Successfully added"
```
Comment (dimigor) [inline Flask_ORM/many_to_many/routes.py:24]: return added staff. Client should know what data were seved

File: Flask_ORM/many_to_many/routes.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+class StaffView(Resource):
+    @marshal_with(staff_structure)
+    def get(self):
+        return StaffModel.query.all()
+
+    def post(self):
+        data = json.loads(request.data)
+        new_staff = StaffModel(**data)
+        db.session.add(new_staff)
+        db.session.commit()
```
Comment (dimigor) [inline Flask_ORM/many_to_many/routes.py:23]: what if data were not saved? catch possible errors

**Review by dimigor — CHANGES_REQUESTED** [review]

File: Flask_ORM/many_to_many/routes.py (hunk 1 above)
Comment (akarev0) [reply Flask_ORM/many_to_many/routes.py:24]: done

File: Flask_ORM/many_to_many/routes.py (hunk 2 above)
Comment (akarev0) [reply Flask_ORM/many_to_many/routes.py:23]: done

**Review by dimigor — APPROVED** [review]

**Pull request merged**